### ISSUE TITLE
GOTCHA: Excel Crash with non-absolute valued properties array

### ISSUE THREAD

**Issue by wanderslth** [flexiodata/flexio-web-app#572]
Something to track as issues end-users may face with their spreadsheet experience.

In this scenario with wikipedia (using gsheets here for screenshot simplicity):

![Untitled spreadsheet - Google Sheets](https://user-images.githubusercontent.com/25911712/68515718-807f9580-0247-11ea-8318-c75498337a77.png)

Formula DOES NOT use absolute positions ($):

`=FLEX("flexiodemo/wikipedia-enrich-people",A2, B1:F1)`

1. Drag down one row.  Excel has the data available in the the previous row, so delivers a blank row as expected.

2.  Drag down all rows.  Excel has no data to work with until the previous rows are calculated.  Excel freezes and requires forced shut down.

Sheets handles this with more grace -- although you can tell by watching the 'loading...' message, that their calculation engine doesn't take calculations in order -- but more 'all at the same time' so you see it working itself out as the 'loading...' bubbles around.  Presume excel works similarly and just doesn't know what to do with the timeout.

TL;DR - people are going to need to be careful when they reference an array of properties.
